ddtrace/mocktracer/mocktracer_test.go

```diff
@@ -74,9 +74,14 @@ func TestTracerStartSpan(t *testing.T) {
 
 func TestTracerFinishedSpans(t *testing.T) {
 	var mt mocktracer
+	assert.Empty(t, mt.FinishedSpans())
+	assert.Empty(t, mt.OpenSpans())
```

Comment:
Please don't squeeze the tests for `OpenSpans` into a test called `TestTracerFinishedSpans`. Instead, write a `TestTracerOpenSpans`

Reply:
added another test